### DISCUSSION TITLE
Turbopack => SassError: Can't find stylesheet to import ?

### DISCUSSION THREAD

**Discussion by WddWade** [vercel/next.js#55627] [App Router]
Server Error
Error: Error evaluating Node.js code
SassError: SassError: Can't find stylesheet to import.
      ╷
    1 │ @import "@/styles/mixins/mixin.scss";
      │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      ╵
      src/components/apps/Leading/WhatWeDo/WhatWeDo.module.scss 1:9  root stylesheet.......

**Comment by 70ki8suda** [comment]
same here
it seems turbo repo can not resolve path alia.

**Comment by Y-k-Y** [comment]
Any updates here? I've faced same issue on 14.1

**Reply by jnm733** [reply]
Same here!

**Reply by alvarlagerlof** [reply]
Perhaps this should be an issue with a reproduction instead.

**Comment by AleksandrKarashev** [comment]
same

**Comment by sooperjos** [comment]
end of 2025 still cant build with turbo or my scss styles are all messed up. This is worse than liquid glass. Hope anyone has a fix for this. not looking forward to refactoring hundreds of files.
